test/mingap_test.cpp

```diff
@@ -80,18 +80,32 @@ TEST(MinGap, SphereSphereOutOfBounds) {
 
 TEST(MinGap, ClosestPointOnEdge) {
   auto a = Manifold::Cube();
-  auto b = Manifold::Sphere(1.0f).Translate({3.0f, 0.0f, 0.5f});
+  auto b = Manifold::Cube().Translate({2.0f, 1.0f, 0.5f});
 
   float distance = a.MinGap(b, 10);
 
-  EXPECT_FLOAT_EQ(distance, 1);
+  EXPECT_FLOAT_EQ(distance, 1.0f);
 }
 
 TEST(MinGap, ClosestPointOnTriangleFace) {
-  auto a = Manifold::Cube().Translate({0.0f, -0.25f, 0.0f});
-  auto b = Manifold::Sphere(1.0f).Translate({3.0f, 0.0f, 0.5f});
+  auto a = Manifold::Cube();
+  auto b = Manifold::Cube()
+               .Scale({10.0f, 10.0f, 10.0f})
+               .Translate({2.0f, -5.0f, -1.0f});
+
+  float distance = a.MinGap(b, 10);
+
+  EXPECT_FLOAT_EQ(distance, 1.0f);
+}
+
+TEST(MinGap, AfterTransformations) {
+  auto a = Manifold::Sphere(1.0f, 100);
+  auto b = Manifold::Sphere(1.0f, 100)
+               .Scale({3.0f, 1.0f, 1.0f})
+               .Rotate(0, 90, 45)
+               .Translate({3.0f, 0.0f, 0.0f});
```

Comment:
Nice test! I don't think all the `.0f` are necessary. I should really clean those up in the rest of our tests.